src/libs/migrations/AddLastVisibleActionCreated.js

```diff
@@ -26,18 +26,18 @@ export default function () {
                     }
 
                     reportsToUpdate[key] = report;
-                    reportsToUpdate[key].lastActionCreated = new Date(report.lastMessageTimestamp)
+                    reportsToUpdate[key].lastVisibleActionCreated = new Date(report.lastMessageTimestamp)
```

Comment:
NAB, but are there any clients who won't have the `lastMessageTimestamp` but would have `lastActionCreated` and not have `lastVisibleActionCreated`. Do we need a migration for that?

Reply:
Good point! We migrated clients from `lastMessageTimestamp` to `lastActionCreated` so I'd expect clients to have `lastActionCreated`, but not have `lastMessageTimestamp`. Additionally, some clients might have `lastActionCreated` but not `lastVisibleActionCreated`. So I think we should update this migration to:

```
reportsToUpdate[key].lastVisibleActionCreated = report.lastActionCreated;
reportsToUpdate[key].lastActionCreated = null;
```

I'd love a buddy check on this logic though.

Reply:
It sounds reasonable to me.

Reply:
Thanks!